packages/react/.storybook/preview.js

```diff
@@ -15,12 +15,16 @@ import {
   Canvas, Story
 } from '@storybook/addon-docs/blocks';
 import { ActionBar, Source } from '@storybook/components';
+
 import prettier from 'prettier/standalone';
 import parserHtml from 'prettier/parser-html';
-import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter';
-import { okaidia } from 'react-syntax-highlighter/dist/esm/styles/prism';
+import parserCss from 'prettier/parser-postcss';
+import SyntaxHighlighter, { Renderer, Wrapper } from './syntax-highlighter';
+
 import '../src/index.scss';
 
+import logo from '!url-loader!@massds/mayflower-assets/static/images/stateseal.png';
```

Comment:
I think we should use the UNPKG CDN for linking the assets (pending on the mayflower-assets release)